Source/Core/VideoCommon/PixelShaderManager.cpp

```diff
@@ -139,11 +139,11 @@ void PixelShaderManager::SetTexDims(int texmapid, u32 width, u32 height)
 {
 	// TODO: move this check out to callee. There we could just call this function on texture changes
 	// or better, use textureSize() in glsl
-	if (constants.texdims[texmapid][0] != 1.0f/width || constants.texdims[texmapid][1] != 1.0f/height)
+	if (constants.texdims[texmapid][0] != 1.0f/128.0f/width || constants.texdims[texmapid][1] != 1.0f/128.0f/height)
 		dirty = true;
 
-	constants.texdims[texmapid][0] = 1.0f/width;
-	constants.texdims[texmapid][1] = 1.0f/height;
+	constants.texdims[texmapid][0] = 1.0f/128.0f/width;
+	constants.texdims[texmapid][1] = 1.0f/128.0f/height;
```

Comment:
Save a divide, do `1.0f / (128.0f * height);` and so on.

BTW, someone will complain about spacing around operators.